fn main() {
    //errors
    

    //panic! its the simplest error handliing


}
